com.jetbrains.Intellij-Community.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Petr Hracek <phracek@redhat.com> -->
+<application>
+    <id type="desktop">com.jetbrains.Intellij-Community.desktop</id>
+    <licence>CC-BY-SA</licence>
```

Comment:
I think this should be

<project_license>Apache-2.0</project_license>
<metadata_license>CC-BY-SA-4.0</metadata_license> (or 3.0, 2.0, 1.0, cf. https://spdx.org/licenses/)

cf.  https://www.jetbrains.com/idea/download/#section=linux